src/models/connection/connection.ts

```diff
@@ -38,19 +40,19 @@ export class Connection {
   }
 
   static getPrimaryServiceDefinition(connection: Connection): SyncedServiceDefinition | undefined {
-    if (connection.firstService.name === 'Redmine') {
+    if (connection.firstService.name === 'Redmine' || connection.firstService.name === 'Jira') {
       return connection.firstService;
-    } else if (connection.secondService.name === 'Redmine') {
+    } else if (connection.secondService.name === 'Redmine' || connection.secondService.name === 'Jira') {
       return connection.secondService;
     } else {
       return undefined;
     }
   }
 
   static getSecondaryServiceDefinition(connection: Connection): SyncedServiceDefinition {
-    if (connection.firstService.name !== 'Redmine') {
+    if (connection.firstService.name === 'Toggl Track') {
       return connection.firstService;
-    } else if (connection.secondService.name !== 'Redmine') {
+    } else if (connection.secondService.name === 'Toggl Track') {
       return connection.secondService;
     } else {
       throw "Cannot occur because two same services cannot be connected"
```

Comment:
Co přesně dělají tyto dvě metody? Přijde mi že obě říkají že RM, Jira, i Toggl mohou být první nebo druhé.

Reply:
Dále zde nějak nevidím Project to project možnost

Reply:
Project 2 project nedá smysl, protože tam primary a secondary není. Secondary umíme zatím jenom Toggl.